packages/suite/src/components/suite/modals/AdvancedCoinSettings/index.tsx

```diff
@@ -44,7 +44,9 @@ const AdvancedCoinSettings = ({ coin, onCancel }: Props) => {
 
     const network = NETWORKS.find(n => n.symbol === coin);
     const [coinInfo, setCoinInfo] = useState<CoinInfo>();
-    const isBlockbook = coinInfo?.blockchainLink?.type === 'blockbook';
+    // TODO: this condition is not correctly applied to Regtest (coinInfo?.blockchainLink === null)
+    // const isBlockbook = coinInfo?.blockchainLink?.type === 'blockbook';
+    const isBlockbook = network?.networkType !== 'ripple';
```

Comment:
@szymonlesisz regarding this `TODO`, should I delete the comments and leave the comments or do you think is there a better way how to do it?

Reply:
just heads up to @vladimirvolek and @slowbackspace 
we are probably going to merge this as it is, so this is going to be another place to take care of I think :thinking:

Reply:
good point, i guess this part of the code will be refactored anyway since there is also upcoming [electrum backend](https://github.com/trezor/trezor-suite/pull/4500) support so there will be other types of backends than blockbook.